plugins/example.yaml

```diff
@@ -3,9 +3,9 @@ kind: Plugin
 metadata:
   name: example
 spec:
-  homepage: https://github.com/seredot/kubectl-example
+  homepage: https://github.com/talos-labs/kubectl-example
   shortDescription: Prints out example manifest YAMLs
-  version: v1.1.0
+  version: v0.0.1
```

Comment:
This has to go up; otherwise existing users can't get the upgrade.

Reply:
Will do!